.github/workflows/test.yml

```diff
@@ -49,9 +70,17 @@ jobs:
           python-version: ${{ matrix.python-version }}
           install-nox: true
 
+      - name: Debug
+        run: |
+          if [ "$github.event.inputs.debug_enabled" == "true" ]; then
+            echo "Debugging is enabled"
+          else
+            echo "Debugging is not enabled"
+          fi
+
       - name: Setup tmate session
         uses: mxschmitt/action-tmate@v3
-        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
+        if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled == 'true' }}
```

Comment:
This corrects a pretty big error I found while testing. In previous conditional debugging would always evaluate to true:

```
if: ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.debug_enabled }}
```

This is because the call returns a string which is truthy. With this, a manual workflow would always be ran in debugging mode because that check was always evaluating to true.

Reply:
Another thing I found when running in debug mode is that the tests would always timeout (execution time > 20 minutes). Should we consider raising the execution timeout if debug mode is enabled?

Reply:
I think that's because of the remote shell created. It stops execution and allows you to connect to the box.

Reply:
Okay, guess the timeout is fine as is then.